model_analyzer/config/generate/optuna_run_config_generator.py

```diff
@@ -44,20 +51,31 @@
 
 logger = logging.getLogger(LOGGER_NAME)
 
+TrialObjectives: TypeAlias = Dict[str, Any]
+ParameterCombo: TypeAlias = Dict[str, Any]
+
 
 class OptunaRunConfigGenerator(ConfigGeneratorInterface):
     """
     Use Optuna algorithm to create RunConfigs
     """
 
+    # This list represents all possible parameters Optuna can currently search for
```

Comment:
What is the plan to take this from a fixed list to full flexibility?

Reply:
I believe this will always be a fixed (ever growing) list because we need to understand/hand-code how we take a parameter and write it into the perf/model/build config we will generate.